src/constants.js

```diff
@@ -2,4 +2,5 @@ export const CHOOSE_CUBE_PAGE = 'CHOOSE_CUBE_PAGE';
 export const GAME_PAGE = 'GAME_PAGE';
 
 export const LIGHT_GRAY = '#d3d3d3';
-export const DARK_GRAY = '#4b4c4c';
+export const DARK_GRAY = 'white';
+export const NOTDARK_GRAY = '#2c2c2c';
```

Comment:
DARK_GRAY should not be white
#2c2c2c looks quite dark

Reply:
Thought it seemed fitting, as far as the visual aspect goes. Would like to load a design of sorts in there because a plain gray square like somewhat like an error screen, but anything can work.

Reply:
The comment was about naming if I want to use dark gray somewhere else I won't expect that it would we white

Reply:
i understand. i was being lazy at the time lol, and thought it'd be funny. should be different now. deleted white entirely and schemed according to light/dark gray

Reply:
i understand. i was being lazy at the time lol. should be different now. deleted white entirely and schemed according to light/dark gray